device/bgp/bgpv4.yaml

```diff
@@ -1,21 +1,39 @@
 components:
   schemas:
-    Device.Bgpv4:
+    Bgp.V4peer:
```

Comment:
V4peer vs. V4Peer?

Reply:
Right, 'V4Peer' is same convention we are following. Thanks.